backend/src/test/java/org/cryptomator/hub/api/VaultResourceTest.java

```diff
@@ -795,8 +794,8 @@ public void testUpdateVaultDespiteLicenseExceeded() {
 
 		@Test
 		@Order(7)
-		@DisplayName("Unlock is blocked if exceeding license seats")
-		public void testUnlockBlockedIfLicenseExceeded() throws SQLException {
+		@DisplayName("Unlock is granted if exceeding license seats but not all users have an token")
```

Comment:
This is not about "not all users have a token".  Maybe: "unlock is granted, if (effective vault user) > license seats but (effective vault user with access token) < license seat". Or something similar.

Reply:
Fixed in https://github.com/cryptomator/hub/pull/258/commits/09b712a82c40fe092a9f8f774f189a96b69d5759